src/components/presentation/NavBar.tsx

```diff
@@ -13,51 +16,77 @@ export interface INavBarProps {
   showBackArrow?: boolean;
 }
 
+const expandIconStyle = { transform: 'rotate(45deg)' };
+
 export const NavBar = (props: INavBarProps) => {
-  const hideCloseButton = useAppSelector((state) => state.formLayout.uiConfig.hideCloseButton);
+  const dispatch = useAppDispatch();
   const language = useAppSelector((state) => state.language.language || {});
-  const theme = useTheme();
+  const { hideCloseButton, showLanguageSelector, showExpandWidthButton, expandedWidth } = useAppSelector(
+    (state) => state.formLayout.uiConfig,
+  );
+
+  const handleExpand = React.useCallback(() => {
```

Comment:
This method should not be necessary to cache in memory, the method is invoked when clicking a button. I can't see any performance improvements by wrapping this within useCallback. 😄